cypress.json

```diff
@@ -4,5 +4,11 @@
   "reporter": "cypress-multi-reporters",
   "reporterOptions": {
     "configFile": "reporter-config.json"
+  },
+  "env": {
+    "openSearchUrl": "http://localhost",
```

Comment:
how about dashboards url?
e.g.
es endpoint is: https://foo.com/
osd endpoint is https://foo.com/_dashbaords

Reply:
we already use the reserved word "baseUrl": "http://localhost:5601" in cypress.json for dashboards url, when we call the cy.visit() api, the baseUrl is added as prefix.

Reply:
The style for os and osd is inconsistent. 1. the variable name. 2. whether port is needed.

Probably use this opportunity to unify the experience. 

```
openSearchUrl
openSearchDashboardsUrl
```

and get rid of the ports. This may require some more changes inside https://github.com/opensearch-project/opensearch-dashboards-test-library/blob/bc5202067ddf73e6bfd98e475e02500647cb1b73/common-utils/network/test-fixture-handler.js#L3

Reply:
For question 1 which to use openSearchDashboardsUrl or baseUrl for variable name:

baseUrl is a reserved word integrated with Cypress framework, quote "URL used as prefix for cy.visit() or cy.request() command's URL". If we decide to create a new env variable like openSearchDashboardsUrl, it loses that prefix function. And we need to rewrite lots of existing test cases, as they are written in cy.visit(relative_path), which implies visiting baseUrl + relative_path. Which I don't suggest to change.

For question 2 whether port is needed:
I think it is necessary to have port configurable. The reason is that local testing, to distinguish frontend and backend, we use port 5601 and 9200. But for remote testing to distinguish frontend and backend, we use relative path like */_dashboard for frontend and */ for backend, and the port is the same (80 for http and 443 for https). So to support both local and remote test, I suggest to keep port configuration. Another tradeoff to note, if we use the baseUrl parameter then the port must be embedded in it. like "baseUrl": "http://localhost:5601", we can't set the port variable separately.

open to more suggestions.

Reply:
For Q2, what I mean is that we can embed port into the endpoint for open search, similar to baseUrl.

Reply:
got it, from test-fixture-handler class, it uses like ${this.hostname}:${this.port} only, we definitely can combine these two variable to one.